client/analytics/components/report-chart/index.js

```diff
@@ -60,6 +61,24 @@ class ReportChart extends Component {
 		const { primary, secondary } = getCurrentDates( query );
 		const primaryKey = `${ primary.label } (${ primary.range })`;
 		const secondaryKey = `${ secondary.label } (${ secondary.range })`;
+		let tooltipValueFormat;
```

Comment:
Not a big deal, but maybe we can extract the `switch` logic into a method to make the render function easier to read?
```suggestion
		const tooltipValueFormat = this.getTooltipValueFormat( selectedChart.type );
```